### ISSUE TITLE
memento endpoint is catching initial requests for webui

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#196]
InterPlanetary Wayback 0.2017.06.15.0114

**Comment by machawk1** [comment]
Does not occur when initially registering SW but error appears on subsequent loads.

![screen shot 2017-06-23 at 11 30 04 am](https://user-images.githubusercontent.com/2514780/27489112-59efdeda-5807-11e7-831d-d8b30be7fee1.png)

**Comment by machawk1** [comment]
Above is not the same issue, which I cannot replicate with a fresh SW. Per above, however, the errors are caused by the AJAX call to openEndedPlaceHolder, which I believe was a placeholder until we resolved the issue of the call to fetch the IPFS web UI link. Errors go away when this call is commented out.

Closing this ticket but will re-open if replicable.
![screen shot 2017-06-23 at 11 34 23 am](https://user-images.githubusercontent.com/2514780/27489350-36d2eb62-5808-11e7-8421-eaba58c9aa7b.png)
